cli/reporter/reporter.go

```diff
@@ -11,6 +12,44 @@ import (
 	"go.mondoo.com/cnquery/shared"
 )
 
+type Format byte
```

Comment:
These changes were also part of https://github.com/mondoohq/cnquery/pull/815 are now in main.

Reply:
I'll rebase it on latest main. Github does not update the references once main changes